src/instructlab/model/chat.py

```diff
@@ -61,6 +62,38 @@
 
 PROMPT_PREFIX = ">>> "
 
+DEFAULT_ENDPOINT = "http://127.0.0.1:8000/v1"
+
+
+def is_openai_server_and_serving_model(endpoint: str) -> bool:
```

Comment:
I think this logic exists somewhere else? I added code a while back to ensure you could only chat w/ a model being served

Reply:
yep: https://github.com/instructlab/instructlab/blob/632df7b83b87060b38eb978e075a12ce03c01446/src/instructlab/model/chat.py#L703-L710

OpenAI compatible servers have a .list() cmd that returns all models in the server

Reply:
Oh good call, I'll update it to use that instead so we aren't duplicating unnecessary code.

Reply:
Updated.